target/linux/mvebu/cortexa9-neon/base-files/etc/uci-defaults/03_wireless

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+#
+# Copyright (C) 2015 OpenWrt.org
+#
+
+[ ! -e /etc/config/wireless ] && exit 0
+
+. /lib/functions.sh
+. /lib/functions/system.sh
+
+board=$(board_name)
+
+case "$board" in
+linksys,caiman|linksys,cobra|linksys,shelby|linksys,venom)
+	SKU=$(strings /dev/mtd3|sed -ne 's/^cert_region=//p')
+	WIFIMAC2G=$(macaddr_add $(cat /sys/class/net/eth0/address) +1)
+	WIFIMAC5G=$(macaddr_add $WIFIMAC2G +1)
+	case "$SKU" in
+		AP)
+			REGD=CN
+		;;
+		AU)
+			REGD=AU
+		;;
+		CA)
+			REGD=CA
+		;;
+		EU)
+			REGD=DE
+		;;
+		US)
+			REGD=US
+		;;
+	esac
+
+	case "$board" in
+		linksys,mamba)
```

Comment:
mamba stays at cortexa9, so this case can be removed, putting the current default case here and using the linksys,mamba case in cortexa9.